### ISSUE TITLE
Generic Metrics: add Exit rate as metric to plot

### ISSUE THREAD

**Issue by Makkmetp** [matomo-org/matomo#15040]
Hi, 
for my dashboard I miss the Exit rate as a metric to plot in de Generics Metrics widget. Is it possible to add this to list of metrics to plot?

**Comment by tsteur** [comment]
BTW which exit rate are you referring to exactly?

**Comment by Makkmetp** [comment]
It would be great for the exit rates of:
- pages
- site search

**Comment by sgiehl** [comment]
But aren't those metrics related to a specific page or site search term? :thinking:

**Comment by Chardonneaur** [comment]
@sgiehl I got a client today who got exactly this need, the kpi they need is the total exit rate at the exit page report.

**Comment by sgiehl** [comment]
@Chardonneaur What is the "total exit rate"? Isn't the exit rate of all visitors always 100% as all visitors exit in the end?
The exit rate is only relevant for a specific page url or search term, as it means the visitors exited afterwards.

**Comment by Chardonneaur** [comment]
![Screenshot from 2022-06-25 06-57-42](https://user-images.githubusercontent.com/28002238/175758810-7bff9698-3d0d-4b20-a4f2-31f6dc5d3bf6.png)
well, to me, in the following screenshot above, it refers to the total that we have here. But on the other hand, I see what you mean, in all cases visitors will end up their visit, so 100% makes sense.
So, I think the big question is mostly what this total average kpi listed on the exit page report mean and why is it under 100%?
@sgiehl : by the way thank you so much for taking the time to read all my comments on Github.

**Comment by sgiehl** [comment]
I'm actually not sure if the totals row for that report makes sense at all. The exit rate is calculated with exits compared to unique page views. But simply summing up the exits and unique page views might not be correct, or at least the resulting values might be meaningless. This might also apply for other action report.
@tsteur what is your opinion on that? Do those total numbers make sense?

**Comment by tsteur** [comment]
The total exit rate lets you see the average exit rate for a page if I see this correctly. This way you can compare if a page is performing better or worse than the average. It be great to learn more how it helps the user or why the metric is important or useful for the user.

**Comment by sgiehl** [comment]
> It be great to learn more how it helps the user or why the metric is important or useful for the user.

Yeah, that be interesting to know. Personally I can't see any benefit in the number. It just says that there's a chance of `X` percent that a user leaves after visiting any page. Maybe @Chardonneaur has more insights on how that number could be useful?
